### DISCUSSION TITLE
OSPOlogy Follow Up: AI Compliance Programs & OSPOs

### DISCUSSION THREAD

**Discussion by anajsana** [todogroup/ospology#366] [OSPO Discussions]
Discuss with the community topics related to the upcoming OSPOlogy meeting:

📅 September 6th ➡️ AI Compliance Programs & OSPOs

* 🕐 Today at [15:00pm UTC | 17:00pm CEST | 8:00am PT](https://www.worldtimebuddy.com/)
* 👩‍💻 [Videoconference Link](https://zoom.us/j/940012604859)
* 💚[Registration page](https://community.linuxfoundation.org/events/details/lfhq-todo-group-ospology-presents-internal-ai-compliance-programs-the-role-of-ospos/) (not mandatory)
* 🙋‍♀️ Panelists: Joanna, Van, Ashley, Georg, Christine
* 📝 [OSPONews Presentation](https://docs.google.com/presentation/d/1UnCKGJBh39YeIsmXL7hZG7pR0L8lfUIDMtlclf0uQX4/edit?usp=sharing) and [Topic of the month presentation](https://docs.google.com/presentation/d/1Irq5OLEOlFc_tXy4nR4e1Uooip9WVSmLnmow2qaVGMs/edit?usp=sharing)

In this open space, people can:

* Ask questions prior to the meeting starts
* Share notes from the conversation
* Add useful links and resources related to the topic
* Connect with the panelists

**Comment by ashleywolf** [comment]
Please feel free to post any questions or comments for us to address during the call!

**Comment by anajsana** [comment]
from webinar platform chat: "Is there a link available for more information around that RFC?"

**Reply by anajsana** [reply]
https://www.federalregister.gov/documents/2023/08/30/2023-18624/artificial-intelligence-and-copyright

**Comment by gioalmeida-ml** [comment]
is there any reference training for developers about a compliant use of generative AIs?

**Reply by VanL** [reply]
I don't think that anyone has a good grasp on what "compliant use" means yet, except to say that it is use that is within the guidelines that your own organization has created.

**Comment by anajsana** [comment]
from webinar platform chat: "The ZF OSPO formed an alliance with the legal/IP team for generative AI topics. We collected developer, cyber security and business development feedbacks and adressed them in a positioning paper, which was published globally"

**Comment by anajsana** [comment]
from webinar platform chat: "In our organization the OSPO reports up through a Tech Innovation function, so there’s a close integration between OSPO and the GenAI work that the innovation team is leading. In fact, the OSPO is bringing open source projects, models, applications to those investigations"

**Comment by anajsana** [comment]
from webinar platform chat "We are especially interested in the challenges of traceability and embedding review guidance for generated items. It should not end up in a blind "copy/paste" process"

**Reply by anajsana** [reply]
"it relates also to commenting guidelines for our developers/labelling generated code"

**Reply by anajsana** [reply]
answer via webinar platform chat: "One thing we are recommending is that any files that include AI-generated content be given just a comment:"

**Comment by anajsana** [comment]
from webinar platform: "What are your experiences @ Van in regards to patent inventor payments, if developers use generated code/docs to file a patent? Is there any information available?"

**Reply by anajsana** [reply]
answer: "The most recent law is that the GAI cannot create the/a point of novelty for a patent, the inventor must be human. That said, there are no restrictions on the use of AI as a part of an invention or even as part of the write-up or spec - you just need to make sure that everything is reviewed because a human needs to affirm to the truth of everything filed."

**Comment by anajsana** [comment]
from the webinar platform chat: "Having worked on SBOM and licensing provenance for years the traceability is a big issue. Forking or copying code is common in open source projects so even if an AI maintains provenance on where it got something it’s maybe wrong. Basically same issue as snippet detection"

**Comment by anajsana** [comment]
from the webinar platform chat: "I think Llama has had the same non-OSI issues - that it has restrictions on what you can use it for"

**Reply by VanL** [reply]
It does. LLaMA has a very liberal license, but it is *not* open source. More on that: https://ospo.co/blog/llama-2-and-open-source/

**Comment by anajsana** [comment]
from the webinar platform chat: "How are you dealing with dataset use?  Do you have policies around that?"

**Comment by anajsana** [comment]
from webinar platform: For anyone interested in a deep dive into some of the legal issues, I recently had this published in the Rutgers Business Law Review: https://papers.ssrn.com/sol3/papers.cfm?abstract_id=4464001
